sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -35,7 +37,7 @@ namespace Azure { namespace Core {
   };
 
   /**
-   * @brief A context is a node within a tree that represents expiration times and key/value pairs.
+   * @brief A context is a node within a tree that represents deadline times and key/value pairs.
```

Comment:
The use of `deadline times` reads strangely. `expiration` was better.

If you really want to keep it, maybe `represents deadlines and ...`